shell/platform/embedder/embedder.h

```diff
@@ -1326,6 +1326,12 @@ typedef void (*FlutterUpdateSemanticsCallback2)(
     const FlutterSemanticsUpdate2* /* semantics update */,
     void* /* user data*/);
 
+typedef void (*FlutterChannelListenedToCallback)(
+    const char* data /* name */,
+    size_t data_len /* name.size */,
```

Comment:
Is a length necessary? I'd assume the name is null terminated?

Reply:
It appears that is a guarantee as of C++11, and I forgot about `c_str`.